src/Applications/LDAS_App/util/inputs/obs_scaling_params/write_netcdf_latlon_grid.m

```diff
@@ -0,0 +1,246 @@
+    netcdf.putAtt(ncid, varid_ll_lat,      'units',         'degrees_north');
+    netcdf.putAtt(ncid, varid_ll_lat,      'axis',          'Y');
+    
+    varid_d_lon      = netcdf.defVar(ncid, 'd_lon',      float_precision, []);
+    netcdf.putAtt(ncid, varid_d_lon,       'standard_name', 'longitude grid spacing');
+    netcdf.putAtt(ncid, varid_d_lon,       'long_name',     'longitude grid spacing');
+    netcdf.putAtt(ncid, varid_d_lon,       'units',         'degrees');
+    netcdf.putAtt(ncid, varid_d_lon,       'axis',          'X');
+    
+    varid_d_lat      = netcdf.defVar(ncid, 'd_lat',      float_precision, []);
+    netcdf.putAtt(ncid, varid_d_lat,       'standard_name', 'latitude grid spacing');
+    netcdf.putAtt(ncid, varid_d_lat,       'long_name',     'latitude grid spacing');
+    netcdf.putAtt(ncid, varid_d_lat,       'units',         'degrees');
+    netcdf.putAtt(ncid, varid_d_lat,       'axis',          'Y');
+    
+    varid_pentad     = netcdf.defVar(ncid, 'pentad',     int_precision,   [dimid_pentad]);
+    netcdf.putAtt(ncid, varid_pentad,      'standard_name', 'pentad');
+    netcdf.putAtt(ncid, varid_pentad,      'long_name',     'pentad');
+    netcdf.putAtt(ncid, varid_pentad,      'units',         '1');
+    netcdf.putAtt(ncid, varid_pentad,      'axis',          'T');
+    
+    varid_start_time = netcdf.defVar(ncid, 'start_time', float_precision, [dimid_pentad]);
+    netcdf.putAtt(ncid, varid_start_time,  'standard_name', 'start time');
+    netcdf.putAtt(ncid, varid_start_time,  'long_name',     'start time');
+    netcdf.putAtt(ncid, varid_start_time,  'axis',          'T');
+    netcdf.putAtt(ncid, varid_start_time,  'units',         'days since 1950-01-01 00:00:00.0 +0000');
+    
+    varid_end_time   = netcdf.defVar(ncid, 'end_time',   float_precision, [dimid_pentad]);
+    netcdf.putAtt(ncid, varid_end_time,    'standard_name', 'end time');
+    netcdf.putAtt(ncid, varid_end_time,    'long_name',     'end time');
+    netcdf.putAtt(ncid, varid_end_time,    'axis',          'T');
+    netcdf.putAtt(ncid, varid_end_time,    'units',         'days since 1950-01-01 00:00:00.0 +0000');
+    
+    varid_lon        = netcdf.defVar(ncid, 'lon',        float_precision, [dimid_lon]);
+    netcdf.putAtt(ncid, varid_lon,         'standard_name', 'longitude');
+    netcdf.putAtt(ncid, varid_lon,         'long_name',     'lower left longitude of gridcell');
+    netcdf.putAtt(ncid, varid_lon,         'units',         'degrees_east');
+    netcdf.putAtt(ncid, varid_lon,         'axis',          'X');
+    
+    varid_lat        = netcdf.defVar(ncid, 'lat',        float_precision, [dimid_lat]);
+    netcdf.putAtt(ncid, varid_lat,         'standard_name', 'latitude');
+    netcdf.putAtt(ncid, varid_lat,         'long_name',     'lower left latitude of gridcell');
+    netcdf.putAtt(ncid, varid_lat,         'units',         'degrees_north');
+    netcdf.putAtt(ncid, varid_lat,         'axis',          'Y');
```

Comment:
@amfox37 : The variable names 'lon' and 'lat' here are not good practice.  They are really 'll_lons' and 'll_lats', but the 'll_*' bit is not reflected in the variable name.  If someone just looks at the nc file and doesn't check the long_name, they might not get this.  
The grid here is a regular lat/lon grid, and we already have the scalar variables 'll_lon' and 'll_lat' in the file.  Together with 'd_lon', 'd_lat', 'N_lon', and 'N_lat' we can readily construct 'll_lons' and 'll_lats'.  
The simplest and safest thing to do is probably to not write 'lon' and 'lat' into the scaling parameter file.  
Alternatively, we could rename the variables to 'll_lons' and 'll_lats'.
Or we could write the grid cell center 'lon' and 'lat' values (i.e., write lon=ll_lons+d_lon/2 and lat=ll_lats+d_lat/2).